certified-connectors/Zoho ZeptoMail/apiDefinition.swagger.json

```diff
@@ -0,0 +1,800 @@
+                      "description": {
+                        "type": "string",
+                        "description": "Mail agent description"
+                      },
+                      "user_connection_status": {
+                        "type": "string",
+                        "description": "User connection status"
+                      },
+                      "mailagent_key": {
+                        "type": "string",
+                        "description": "Mail agent key"
+                      },
+                      "status": {
+                        "type": "string",
+                        "description": "status"
+                      }
+                    }
+                  },
+                  "description": "data",
+                  "minLength": 1
+                },
+                "message": {
+                  "type": "string",
+                  "description": "message"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get mail agent",
+        "description": "Get mail agent",
+        "operationId": "get_mail_agent",
+        "x-ms-visibility": "internal",
+        "parameters": []
+      }
+    },
+    "/portal/v1.0/mailagents/{mailagent_key}/bounce-address": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "bounce_address": {
+                        "type": "string",
+                        "description": "Bounce address"
+                      },
+                      "is_default": {
+                        "type": "boolean",
+                        "description": "Is default"
+                      }
+                    }
+                  },
+                  "description": "data"
+                },
+                "message": {
+                  "type": "string",
+                  "description": "message"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get bounce address",
+        "description": "Get bounce address",
+        "x-ms-visibility": "internal",
+        "operationId": "get_bounce_address",
+        "parameters": [
+          {
+            "name": "mailagent_key",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "mail agent key",
+            "x-ms-summary": "mail agent key",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    },
+    "/v1.0/email": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {}
```

Comment:
@zohomail empty schema is not allowed please remove or define a proper schema. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).
Please fix all empty schemas in the swagger file.